src/compiler/python_generator.cc

```diff
@@ -232,10 +232,14 @@ static void PrintAllComments(const DescriptorType* desc, Printer* printer) {
 bool PrintBetaServicer(const ServiceDescriptor* service,
                        Printer* out) {
   out->Print("\n\n");
-  out->Print("class Beta$Service$Servicer(object):\n", "Service",
-             service->name());
+  out->Print("class Beta$Service$Servicer(object):\n",
```

Comment:
The changes on these two lines (and on similar lines elsewhere) appear to have no effect and are unrelated to the semantic change of this commit. Accidental edits?